guides/git-workflow.md

```diff
@@ -25,21 +93,21 @@ Any changes that you want to eventually make to the `master` branch of the `spac
 
 5. Push the branch to the GitHub repository for the personal fork with `git push origin <branchname>`.
 
-6. On the `spacetelescope` repository, create a pull request for the recently pushed branch.  You will want to set the base fork pointing to `spacetelescope:master` and the `head` fork pointing to the branch on your personal fork (i.e. `username:branchname`).  Note that if the branch is still under heavy development, you can put `WIP:` at the beginning of the pull request title to signify that the pull request is still a work in progress (i.e. `WIP: Example Pull Request Title`).  Not until the `WIP:` tag is explicitly removed will the pull request be deemed 'mergeable'.
+6. In the `spacetelescope` repository, create a pull request for the recently pushed branch.  You will want to set the base fork pointing to `spacetelescope:master` and the `head` fork pointing to the branch on your personal fork (i.e. `username:branchname`).  Note that if the branch is still under heavy development, you can put `WIP:` at the beginning of the pull request title to signify that the pull request is still a work in progress (i.e. `WIP: Example Pull Request Title`).  Not until the `WIP:` tag is explicitly removed will the pull request be deemed 'mergeable'.
 
 7. Assign the pull request a reviewer, selecting a maintainer of the `spacetelescope` repository.  They will review your pull request and either accept the request and merge, or ask for additional changes.
 
 8. Iterate with your reviewer(s) on additional changes if necessary, addressing any comments on your pull request.  If changes are required, you may end up iterating over steps 4.ii, 4.iii and 5 several times while working with your reviewer.
 
 9. Once the pull request has been accepted and merged, you can delete your local branch with `git branch -d <branchname>`.
 
-### Keeping your fork updated
+#### Keeping your fork updated <a name="update-fork"></a>
```

Comment:
I feel like this also applies to branching workflow. 🤷

Reply:
Yeah there are many steps that are the same between each work flow. 

This might take a little work with the markdown syntax to pull off, but I can imagine presenting this all as a two column table instead. Left hand column for branching, right hand column for forking. And steps that are identical between the two workflows span both columns.

Reply:
This sounds like a great improvement in a future PR! 😉

Reply:
But yes, I will add some instruction here on how to keep a branch updated as well.